index.html

```diff
@@ -208,23 +212,64 @@ <h4 class="item_name">Frameworks</h4>
         </ul>
       </li>
       <li class="other_item">
-        <img class="item_logo" src="assets/skills.png" alt="language logo"/>
+        <img class="item_logo" src="assets/skills.png" alt="language logo" />
         <h4 class="item_name">Skills</h4>
         <ul class="items_sub_list">
           <li class="items_sub">Codekit</li>
           <li class="items_sub">GitHub</li>
           <li class="items_sub">Codepen</li>
-          
+
           <li class="items_sub">Gitlab</li>
           <li class="items_sub">Terminal</li>
         </ul>
       </li>
     </ul>
-    <section class="contact_me_section">
-      <div class="contact_text">
+    <footer class="portofolio_footer">
+      <div class="contact_me_section">
+        <h1 class="contact_form_title">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h1>
+        <form class="contact_form" action="https://formspree.io/f/mknegjnj" method="POST">
+          <input type="text" class="form_input" placeholder="Full Name" maxlength="30" required>
+          <input type="email" class="form_input" placeholder="Email address" required>
+          <textarea id="contact_text_area" class="form_input" placeholder="write your message here" maxlength="500"
+            required></textarea>
+          <button class="submit_button" type="submit">Get in touch</button>
+        </form>
```

Comment:
- [ ] Please style your button interations such as hover, active and enabled according to the Figma template you have chosen.
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/186276840-3cf88acf-1b7c-40ed-bbc5-7ff175b056b3.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/186277105-0c2f05e1-9d69-4164-80c5-793b0df6c0f4.png)

Reply:
thank you